packages/x-charts/package.json

```
@@ -75,7 +75,7 @@
     "./*": {
       "types": "./*/index.d.ts",
       "require": "./*",
```

I don't know exactly how to verify this, but would this line suffer from the same problem? e.g. should this be


```suggestion
      "require": "./*/index.js",
```

I'm not 100% sure and I can't find it immediately in the docs, but I believe package exports don't follow Node.js module resolution rules. i.e. extensionless paths or folder index.js are not expected to resolve in all environments.